frontend/src/actions/skew.js

```diff
@@ -7,13 +7,35 @@ import { getHistory } from './history';
 
 export const SET_UNLABELED_DATA = 'SET_UNLABELED_DATA';
 export const SET_LABEL_COUNTS = 'SET_LABEL_COUNTS';
+export const SET_SKEW_PAGES = 'SET_SKEW_PAGES';
 
 export const set_unlabeled_data = createAction(SET_UNLABELED_DATA);
 export const set_label_counts = createAction(SET_LABEL_COUNTS);
+export const set_skew_pages = createAction(SET_SKEW_PAGES);
 
 //Get the data for the skew table
-export const getUnlabeled = (projectID) => {
-    let apiURL = `/api/data_unlabeled_table/${projectID}/`;
+export const getUnlabeled = (queryObj, projectID) => {
+
+    let queryURLObj = new URLSearchParams("");
+
+    queryURLObj.append("project", projectID);
+    queryURLObj.append("page", queryObj.page + 1);
+
+    if (queryObj.filtered.length > 0) {
+        queryURLObj.append("text__icontains", queryObj.filtered[0].value);
+    }
+
+    if (queryObj.sorted.length > 0) {
+        if (queryObj.sorted[0].desc) {
+            queryURLObj.append("ordering", "text");
+        } else {
+            queryURLObj.append("ordering", "-text");
+        }
+
+    }
+
+
+    let apiURL = `/api/data_unlabeled_table/?${queryURLObj.toString()}`;
```

Comment:
I'm not sure if you are able to add a debounce to the search filter for this or not.  I think its a nice-to-have but not necessary if time is an issue.

Reply:
I don't see an obvious way to do this. I think this just uses onChange for an input so there might not be a concept of "finished typing".

Reply:
So I added a change in 1ff2926 so that the search only happens when you hit enter. I think this should address the main issue. Thoughts?

Reply:
I think I'd prefer it the original way. Since there is no submit button its not clear that pressing enter will start the search.  Also for existing users it directly changes a feature that they are already used to.